collateral/thresholds.json

```diff
@@ -0,0 +1,21 @@
+{
+	"genome" :	{
+				"index"				:	"0",
```

Comment:
I'd prefer yaml here for sane type inference (don't need to write floats as strings then read strings as floats)

Reply:
Tomato, tomahto :) What I would like to do is make it support either YAML or JSON, but I don't think that's important now